tests/operators/test_operators.py

```diff
@@ -134,6 +134,62 @@ def test_sum_operator_linearoperator():
     assert not isinstance(c, LinearOperator), 'Operator + LinearOperator should not be a LinearOperator'
 
 
+def test_sum_operator_tensor():
+    rng = RandomGenerator(0)
+    a = DummyOperator(torch.tensor(3.0))
+    b = rng.complex64_tensor((3, 10))
+    c = a + b
+    x = rng.complex64_tensor(10)
+    (y1,) = c(x)
+    y2 = a(x)[0] + b * x
+
+    torch.testing.assert_close(y1, y2)
+    assert isinstance(c, Operator), 'Operator + tensor should be an Operator'
+    assert not isinstance(c, LinearOperator), 'Operator + tensor should not be a LinearOperator'
+
+
+def test_rsum_operator_tensor():
+    rng = RandomGenerator(0)
+    a = DummyOperator(torch.tensor(3.0))
+    b = rng.complex64_tensor((3, 10))
+    c = b + a
+    x = rng.complex64_tensor(10)
+    (y1,) = c(x)
+    y2 = a(x)[0] + b * x
+
+    torch.testing.assert_close(y1, y2)
+    assert isinstance(c, Operator), 'Operator + tensor should be an Operator'
+    assert not isinstance(c, LinearOperator), 'Operator + tensor should not be a LinearOperator'
+
+
+def test_sum_linearoperator_tensor():
+    rng = RandomGenerator(0)
+    a = DummyLinearOperator(rng.complex64_tensor((3, 3)))
+    b = rng.complex64_tensor((3,))
+    c = a + b
+    x = rng.complex64_tensor(3)
+    (y1,) = c(x)
+    y2 = a(x)[0] + b * x
+
+    torch.testing.assert_close(y1, y2)
+    assert isinstance(c, LinearOperator), 'LinearOperator + tensor should be a LinearOperator'
+    assert isinstance(c, Operator), 'LinearOperator + tensor should be an Operator'
```

Comment:
This of course does not hurt but it is not really provide additional information, does it?